File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/FameRankJson.java
package at.hannibal2.skyhanni.data.jsonobjects.repo;

import com.google.gson.annotations.Expose;

import java.util.Map;

public class FameRankJson {
    @Expose
    public Map<String, FameRank> fame_rank;

    public static class FameRank {
        @Expose
        public String name;

        @Expose
        public int fame_required;

        @Expose
        public double bits_multiplier;

        @Expose
        public int votes;
    }
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/FameRankJson.kt
package at.hannibal2.skyhanni.data.jsonobjects.repo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class FameRankJson(
    @Expose @SerializedName("fame_rank") val fameRank: Map<String, FameRank>
)

data class FameRank(
    @Expose val name: String,
    @Expose @SerializedName("fame_required") val fameRequired: Int,
    @Expose @SerializedName("bits_multiplier") val bitsMultiplier: Double,
    @Expose val votes: Int
)
